parent/pom.xml

```diff
@@ -275,6 +275,8 @@
                 <artifactId>apache-rat-plugin</artifactId>
                 <configuration>
                     <excludes combine.children="append">
+                        <exclude>**/rat.txt</exclude>
```

Comment:
I thought this was excluded by default. I have never gotten an error regarding `rat.txt`

Reply:
I would assume so as well. Especially since those files are in `target`.